app/Models/CategoryDAO.php

```diff
@@ -77,6 +77,15 @@ public function deleteCategory($id) {
 		}
 	}
 
+	public function select() {
+		$sql = 'SELECT id, name FROM `' . $this->prefix . 'category`';
+		$stm = $this->bd->prepare($sql);
+		$stm->execute();
```

Comment:
I will make another PR, which more appropriately uses `exec()` vs. `query()` vs. `prepare() + execute()`, and makes a uniform use of `ATTR_EMULATE_PREPARES` (currently off for MySQL / MariaDB, but on for SQLite and PostgreSQL)